dispatcher.go

```diff
@@ -276,6 +294,7 @@ func (d *Dispatcher) Start() error {
 		allStarted []transport.Lifecycle
 	)
 
+	d.log.Info("Starting up.")
```

Comment:
I don't have strong opinions on log formatting -- I want to make sure the logs are useful (concise but clear), but don't mind otherwise.

When pouring through logs, the sentences can help you easily find the beginning/end of a log message. However, when searching for logs, it is a little annoying to remember to capitalize and add a period when trying to match to an exact log message.